docker/build_geant4.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+source ${docker_env}
+
+rm -rf ${geant4_build_dir}/bld ${geant4_install_dir}
+mkdir -p ${geant4_build_dir}/bld
+cd ${geant4_build_dir}
+wget http://geant4.cern.ch/support/source/geant4.${geant4_version}.tar.gz
```

Comment:
2 quick comments regarding security for downloading `tar.gz` files:

1. If `geant4.cern.ch` offers `https`, please prefer using that over standard `http`.
2. If `geant4.${geant4_version}.tar.gz` offers a checksum for validating the file, you should check that checksum here.

Both here and throughout.